src/main/resources/migrations/00022_alter_table_invites_add_column_disabled.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:alter-table-invites-add-column-disabled
+ALTER TABLE invites ADD COLUMN disabled BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Once again you have to be careful here making `DEFAULT FALSE`. Coz in case if you had live invites in prod DB already, this would have made all of them active. However, we are safe here coz there's no live invites.